test/integration/targets/azure_rm_recoveryservicesvault/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+- name: Create or Update Recovery Services vault
+  azure_rm_recoveryservicesvault:
+    resource_group: "{{ resource_group }}"
+    name: testVault
+    sku:
+      name: Standard
+    location: "{{ location }}"  
+  register: output
+
```

Comment:
also run the same task again to check if nothing changes,

and then try to change one of the parameters, I wonder if we can change for instance SKU